pontoon/base/tests/test_helpers.py

```diff
@@ -54,3 +60,49 @@ def test_unicode(self):
 
     def test_escape(self):
         assert_equal(str(nospam('<>\'"@&')), '&lt;&gt;&quot;&quot;&#64;&amp;')
+
+
+class AsSimpleTranslationTests(TestCase):
+    """Tests related to the as_simple_translation template filter."""
+
+    def test_empty_string(self):
+        source = ''
+        assert_equal(as_simple_translation(source), '')
```

Comment:
What if `source` is `None`?

Reply:
The filter is used on `Translation.string`, which is never None.

Should I add protection anyways?

Reply:
Nope, I just wanted to know.